cloud/migrate-mst-cloud/schema-then-data.md

```diff
@@ -0,0 +1,336 @@
+# Migrate schema and data separately
+Migrate larger databases by migrating your schema first, then migrating the
+data. This method copies each table or chunk separately, which allows you to
+restart midway if one copy operation fails.
+
+The procedure to migrate your database requires these steps:
+*   [Migrate schema pre-data](#migrate-schema-pre-data)
+*   [Restore hypertables in Timescale Cloud](#restore-hypertables-in-timescale-cloud)
+*   [Copy data from Managed Service for TimescaleDB](#copy-data-from-managed-service-for-timescaledb)
+*   [Restore data into Timescale Cloud](#restore-data-into-timescale-cloud)
+*   [Migrate schema post-data](#migrate-schema-post-data)
+*   [Recreate continuous aggregates](#recreate-continuous-aggregates) (optional)
+*   [Recreate policies](#recreate-policies) (optional)
+*   [Recompress previously compressed data](#recompress-data) (optional)
```

Comment:
```suggestion
*   [Migrate schema pre-data]: #migrate-schema-pre-data
*   [Restore hypertables in Timescale Cloud]: #restore-hypertables-in-timescale-cloud
*   [Copy data from Managed Service for TimescaleDB]: #copy-data-from-managed-service-for-timescaledb
*   [Restore data into Timescale Cloud]: #restore-data-into-timescale-cloud
*   [Migrate schema post-data]: #migrate-schema-post-data
*   [Recreate continuous aggregates]: #recreate-continuous-aggregates
*   [Recreate policies]: #recreate-policies
*   [Recompress previously compressed data]: #recompress-data
```

Reply:
Lana, how did you want the final page to look with this edit? This way seems to break Gatsby: 
<img width="662" alt="image" src="https://user-images.githubusercontent.com/26616127/153267731-684e1bca-fa7e-498f-81be-8d5e855a19c3.png">

Reply:
Oh, I'm an idiot 🤦‍♀️  Delete this suggestion, sorry. It was late in the day, clearly!

Reply:
Happens to us all 😄